andela/mmdp_site
# MMDP-552 Fix User is unable to upload a research document on the 'Research' section of the Content Management System until the page is reloaded.

**What does this PR do?**

Fixes User is unable to upload a research document on the 'Research' section of the Content Management System until the page is reloaded.

**How should this be manually tested?**

- `git fetch`
- `git checkout bg-fix-upload-research-doc-MMDP-552` 
- `yarn run dev-server`
- navigate to the path on the CMS: `/resources/research/all`
- add new research. After success message has been returned,try adding new research again. Expect to be redirected to page where you can upload new research.

**What are the relevant Jira issues?**

[MMDP-552](https://viisaus.atlassian.net/browse/MMDP-552)
